sources/packages/web/src/components/students/applicationTracker/Completed.vue

```diff
@@ -250,6 +284,8 @@ export default defineComponent({
     return {
       assessmentDetails,
       multipleCOEDenialReason,
+      ecertFailedValidationDetails,
+      ECERT_FAILED_MESSAGES,
```

Comment:
There is no apparent reason for `ECERT_FAILED_MESSAGES` to be returned in the setup. Please let me know if I am missing something.